helm/grafana/values.yaml

```diff
@@ -1,3 +1,7 @@
+global:
+  imageRegistry: null
```

Comment:
Sorry to be a pain again but I think you need to regenerate the schema and also run the sed command in the Contributing.md file to ensure null types can be strings ("type": ["string", "null"] instead of "type": "null")

Reply:
Also maybe we can set docker.io here directly and then remove the registry lines below ?

Reply:
yeah I already regenerated the schema but forgot to do it on the last commits.

> run the sed command in the Contributing.md file to ensure null types can be strings ("type": ["string", "null"] instead of "type": "null")

Which command are you reffering to ?

Reply:
And moreover, even with the last commits, I still get errors that shouldn't appear anymore

Reply:
Oh [this one](https://github.com/giantswarm/loki-app/blob/master/CONTRIBUTING.md#upgrading) but that's only in loki :(

Reply:
Can you set   testFramework:
    enabled: false in the values file? That should fix your issue :)

Reply:
It's ok I used it anyway

Reply:
The issue is here       image: "docker.io/giantswarm/bats":v1.4.1"

Reply:
Oh you have an extra quote

Reply:
I merged a suggestion to fix it, don't forget to pull :)